src/cairo_runner.rs

```diff
@@ -0,0 +1,201 @@
+use crate::{relocatable::PyRelocatable, utils::to_py_error, vm_core::PyVM};
+use cairo_rs::{
+    cairo_run::write_output,
+    hint_processor::builtin_hint_processor::builtin_hint_processor_definition::BuiltinHintProcessor,
+    types::{program::Program, relocatable::Relocatable},
+    vm::{
+        errors::{
+            cairo_run_errors::CairoRunError, runner_errors::RunnerError, trace_errors::TraceError,
+        },
+        runners::cairo_runner::CairoRunner,
+    },
+};
+use num_bigint::{BigInt, Sign};
+use pyo3::prelude::*;
+use std::{
+    collections::HashMap,
+    path::{Path, PathBuf},
+};
+
+#[pyclass(unsendable)]
+#[pyo3(name = "CairoRunner")]
+pub struct PyCairoRunner {
+    inner: CairoRunner,
+    pyvm: PyVM,
+    hint_processor: BuiltinHintProcessor,
+    hint_locals: HashMap<String, PyObject>,
+}
+
+#[pymethods]
+impl PyCairoRunner {
+    #[new]
+    pub fn new(path: String, entrypoint: String) -> PyResult<Self> {
+        let program = Program::new(Path::new(&path), &entrypoint).map_err(to_py_error)?;
+        let cairo_runner = CairoRunner::new(&program).map_err(to_py_error)?;
+
+        Ok(PyCairoRunner {
+            inner: cairo_runner,
+            pyvm: PyVM::new(
+                BigInt::new(Sign::Plus, vec![1, 0, 0, 0, 0, 0, 17, 134217728]),
```

Comment:
This prime should be taken from the compiled program. 
I will create a issue about this